src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.List;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private FruitDao fruitDao = new FruitDaoImpl();
+    private List<String> testList;
+
+    @Test(expected = RuntimeException.class)
+    public void putNullList_notOk() {
+        testList = null;
+        fruitDao.putDataIntoStorage(testList);
+    }
+
+    @Test
+    public void putList_Ok() {

```

Comment:
For method namings use such convention: "methodUnderTest_state_expectedBehavior"

Reply:
```suggestion
    public void putDataIntoStorage_putList_Ok() {

```